index.js

```diff
@@ -74,6 +53,16 @@ class BabelCompiler {
     this.options.filename = file.path;
     this.options.sourceFileName = file.path;
 
+    if (!this.options.presets) {
+      const babelConfig = new babel.OptionManager().init(this.options);
+      console.log(babelConfig);
+      const hasConfig = babelConfig.presets && babelConfig.presets.length > 0;
+
+      if (!hasConfig) {
+        this.options.presets = ['latest'];
```

Comment:
@denysdovhan I think no defaults is the safest way 😔. If we will ignore even ie, I predict a lot of issues with runtime errors.